wp-content/themes/borderzine/functions.php

```diff
@@ -45,4 +45,38 @@ function borderzine_stylesheets() {
 	);
 
 }
-add_action( 'wp_enqueue_scripts', 'borderzine_stylesheets', 20 );
\ No newline at end of file
+add_action( 'wp_enqueue_scripts', 'borderzine_stylesheets', 20 );
+
+/**
+ * The Borderzine sidebar shortcode function
+ *
+ * @param Array $atts Shortcode attributes or block properties.
+ * @param String $content Shortcode wrapped text; not used in this shortcode.
+ * @param String $tag The complete shortcode tag; not used in this shortcode.
+ * @return HTML
+ * @since 0.1
+ */
+function borderzine_sidebar_shortcode( $atts = array(), $content = '', $tag = '' ) {
+	// This is the shortcode that disables doing shortcodes.
+	if ( isset( $atts['no'] ) ) {
+		return '';
+	}
```

Comment:
This piece isn't needed. In SCAIP it told the automatic inserter to disable itself, but this doesn't have an automatic inserter.